src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -175,12 +173,18 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         return this._store.dispatch(this._paymentStrategyActionCreator.widgetInteraction(() => {
             return this._googlePayPaymentProcessor.displayWallet()
                 .then(paymentData => this._paymentInstrumentSelected(paymentData))
-                .then(() => onPaymentSelect())
+                .then(() => {
+                    if (onPaymentSelect) {
```

Comment:
`onPaymentSelect` has a default value `() => {}` (a no-op function) so you shouldn't need to do a null check.